src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -53,8 +53,8 @@ case class InitializationFileException(googleProject: GoogleProject, clusterName
 case class BucketObjectException(gcsUri: String)
   extends LeoException(s"The provided GCS URI is invalid or unparseable: ${gcsUri}", StatusCodes.BadRequest)
 
-case class BucketObjectAccessException(userEmail: WorkbenchEmail, gcsUri: GcsPath)
-  extends LeoException(s"${userEmail.value} does not have access to ${gcsUri.toUri}", StatusCodes.Forbidden)
+case class BucketObjectAccessException(userEmail: Option[WorkbenchEmail], gcsUri: GcsPath, errorMessage: String)
```

Comment:
Is this relevant to this PR? I feel like this was maybe carried over from a different change? Should probably revert it.